src/components/analyses/useAnalysisRunTime.js

```diff
@@ -26,15 +27,17 @@ function useAnalysisRunTime(
     stepFilterFn = (step) => step.step_number === 1
 ) {
     const isRunning = analysis?.status === analysisStatus.RUNNING;
+    const isComplete = analysis?.status === analysisStatus.COMPLETED;
```

Comment:
I went with just completed as opposed to terminated (which includes cancelled, failed, and removed) because I wasn't sure if there would be interest in the run time for all those states, but let me know if others should be added.

Reply:
I vote to add this feature for all terminal states. 👍🏽